src/typeorm/entity/ward.entity.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  OneToMany,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { District } from './district.entity';
+import { User } from './user.entity';
+
+@Entity('wards')
+export class Ward {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  name: string;
+
+  @Column({
+    type: 'timestamp',
+  })
+  create_at: Date;
+
+  @Column({
+    type: 'timestamp',
+  })
+  update_at: Date;
+
+  @Column({
+    type: 'timestamp',
+  })
+  delete_at: Date;
+
+  @Column()
+  district_id: number;
+
+  @ManyToOne(() => District, (district) => district.wards, {
+    onDelete: 'SET NULL',
+  })
+  @JoinColumn({ name: 'district_id' })
+  district: District;
```

Comment:
district_id type not null thì làm sao có thể SET NULL dc?
Nếu đang dùng deleted_at (soft delete) thì district_id NO_ACTION hoặc RESTRICT

Thứ 2, Đang mâu thuẫn với migration: CASCADE